pom.xml

```diff
@@ -587,7 +587,6 @@ under the License.
                 <groupId>org.apache.accumulo</groupId>
                 <artifactId>accumulo-minicluster</artifactId>
                 <version>${accumulo.version}</version>
-                <scope>test</scope>
```

Comment:
this was a copy paste problem from merger, a ticket has already been made for the problem in merger and it will be the same fix.